<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>系统管理</string>
    </property>
    <addaction name="action_login"/>
    <addaction name="separator"/>
    <addaction name="action_connect"/>
    <addaction name="action_disconnect"/>
    <addaction name="action_export"/>
    <addaction name="separator"/>
    <addaction name="action_index"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>运动员管理</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>赛事管理</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_login">
   <property name="text">
    <string>登录</string>
   </property>
   <property name="toolTip">
    <string>login</string>
   </property>
  </action>
  <action name="action_connect">
   <property name="text">
    <string>连接</string>
   </property>
   <property name="toolTip">
    <string>connect</string>
   </property>
  </action>
  <action name="action_disconnect">
   <property name="text">
    <string>断开</string>
   </property>
   <property name="toolTip">
    <string>disconnect</string>
   </property>
  </action>
  <action name="action_export">
   <property name="text">
    <string>导出</string>
   </property>
   <property name="toolTip">
    <string>export</string>
   </property>
  </action>
  <action name="action_index">
   <property name="text">
    <string>主页</string>
   </property>
   <property name="toolTip">
    <string>index</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
